config/smart_answers/marriage_abroad_data.yml

```diff
@@ -185,10 +184,12 @@ countries_with_18_outcomes:
   - uzbekistan
   - vanuatu
   - venezuela
-  - vietnam
   - western-sahara
   - yemen
   - zambia
+countries_with_18_outcomes_consular_civil_partnerships:
```

Comment:
This threw me a bit - I understand the 18 outcomes refer to the `_opposite_sex`/`_same_sex` files in each of the `partner_british`/`partner_local`/`partner_other` directories in each of the `ceremony_country`/`third_country`/`uk` directories (2*3*3=18).

We're now adding 3 more outcomes for Japan and Panama, and 2 more outcomes for Vietnam, [following the logic](https://github.com/alphagov/smart-answers/pull/5507#issuecomment-897416005). Should this particular subset be renamed `countries_with_21_outcomes`?

Reply:
> Thanks @beccapearce - I vaguely remember that Slack conversation now 👍 probably worth explaining in the commit message why we've gone a different route. I do think the new outcomes should be moved down one level, though, into `ceremony_country`, as the outcomes only apply in that flow.

Hm, I think if we move the outcome down a level then we need to replicate it across the other levels as well. I believe this is an outcome for `ceremony_country` as well as `third country` and `partner_local` that would be 9 places to add the outcome for panama, 6 for vietnam and then the one place for japan.

Reply:
Would it be ok to name them countries_with_21_outcomes? I think Panama would have 27 and Vietnam would have 24 (if I counted correctly)

Reply:
Perhaps `countries_with_multiple_outcomes_including_consular_civil_partnerships`?

Reply:
Hmmm if we keep the file structure as it is currently then we can do 19 outcomes. It's only a problem if we move the outcome files within the subfolders

Reply:
That's true. Let's do that 👍  `countries_with_19_outcomes`?